google/resource_compute_autoscaler_test.go

```diff
@@ -221,19 +221,14 @@ func testAccCheckComputeAutoscalerUpdated(n string, max int64) resource.TestChec
 
 func testAccComputeAutoscaler_scaffolding(it_name, tp_name, igm_name string) string {
 	return fmt.Sprintf(`
-data "google_compute_image" "my_image" {
```

Comment:
Oh no, looks like a different PR in terraform modified tests that get copied over and so this is trying to overwrite them. Would you mind just doing these changes too in your PR so that we don't have to worry about merges and stuff?

Reply:
Happy to, although we should figure out who wrote that PR and let them know they should be doing those changes in MM for this resource going forward :)